Gulpfile.ts

```diff
@@ -748,7 +745,7 @@ gulp.task("runtests",
 const nodeServerOutFile = "tests/webTestServer.js";
 const nodeServerInFile = "tests/webTestServer.ts";
 gulp.task(nodeServerOutFile, false, [servicesFile], () => {
-    const settings: tsc.Settings = getCompilerSettings({}, /*useBuiltCompiler*/ true);
+    const settings: tsc.Settings = getCompilerSettings({module: "commonjs"}, /*useBuiltCompiler*/ true);

```

Comment:
I'm not sure what whitespace you're complaining about, but whatever it is I guess it should be codified because our lint rules don't catch it.